custom_components/deebot/number.py

```diff
@@ -80,6 +80,18 @@ def _volume_icon(instance: "DeebotNumberEntity") -> str | None:
         native_step=1.0,
         icon="mdi:counter",
     ),
+    DeebotNumberEntityDescription(
+        capability_fn=lambda caps: caps.clean.pads.interval,
```

Comment:
this one should be `capability_fn=lambda caps: caps.clean.pads_cleaning_interval`